jcl/src/openj9.dtfj/share/classes/com/ibm/jvm/j9/dump/extract/Main.java

```diff
@@ -367,20 +353,18 @@ private static void ensure(boolean condition, String errorMessage) {
 	}
 
 	private static void usageMessage(String message, int code) {
-		report("Usage: jextract dump_name [output_filename] [options]"); //$NON-NLS-1$
-		report(" output filename defaults to dump_name.zip"); //$NON-NLS-1$
-		report(" options:"); //$NON-NLS-1$
-		report("   -f executable_name override executable name"); //$NON-NLS-1$
-		report("   -help              print this usage message"); //$NON-NLS-1$
-		report("   -v                 enable verbose output"); //$NON-NLS-1$
-		if (_throwExceptions) {
-			throw new JExtractFatalException(message, code);
-		} else {
-			if (message != null) {
-				report(message);
-			}
-			System.exit(code);
-		}
+		report("Usage: jextract [options] dump_name [output_filename]"); //$NON-NLS-1$
+		report("  output filename defaults to dump_name.zip"); //$NON-NLS-1$
+		report("  options:"); //$NON-NLS-1$
+		report("    -e                  throw exceptions instead of calling System.exit()"); //$NON-NLS-1$
+		report("    -f executable_name  override executable name"); //$NON-NLS-1$
+		report("    -help               print this usage message"); //$NON-NLS-1$
+		report("    -interactive        start an interactive session"); //$NON-NLS-1$
```

Comment:
This is no longer useful, it was replaced by DDR. We kept it alive after DDR was available to support the trprint subcommand, but trying to run that, it doesn't seem to work very well any more and I don't think anybody uses it. Bottom line, it shouldn't be included in the help, and I expect we could delete all the associated code.

Reply:
Ok, I won't mention `-interactive`. I created #10764 to track removing the option and the support infrastructure.